content/version/2/0/code_of_conduct.nl.md

```diff
@@ -0,0 +1,93 @@
++++
+version = "2.0"
+aliases = ["/version/2/0"]
++++
+
+# Contributor Convenant Gedragscode (Code of Conduct)
+
+## Onze belofte
+
+Wij als leden, deelnemers en leiders beloven om deelname aan onze community een intimidatievrije ervaring te maken voor iedereen, ongeacht leeftijd, lichaamsbouw, zichtbare of onzichtbare handicap, etniciteit, geslachtskenmerken, genderidentiteit en expressie, ervaringsniveau, opleiding, sociaaleconomische status, nationaliteit, persoonlijk voorkomen, ras, religie of seksuele identiteit en oriëntatie.
```

Comment:
I think that "deelnemers" is not quite right as a translation for "contributors". I would favor something else like "inzenders".

Reply:
@brodybits what about "bijdragers", like in 1.4? https://github.com/ContributorCovenant/contributor_covenant/blob/release/content/version/1/4/code-of-conduct.nl.md
Also I now notice a trailing "n" in Contributor Covenant.

Reply:
"deelnemers" feels like an appropriate translation to me

Reply:
Contribuanten would be the correct wording

Reply:
"bijdragers" sounds fine for me ("deelnemers" made me think of "shareholders" or "participants")

"contribuanten" (probably should add another "t" --> "contributanten") sounds more clear to me, though I don't know how proper it is (I left Amsterdam almost 4 years ago)

Reply:
leiders should be consistent further down it's called Gemeenschapsleiders - perhaps thats a better definition

Reply:
Contributant (with extra t) is French, not dutch - in dutch it would be contribuanten https://www.encyclo.nl/begrip/contribuant

Reply:
In general: I'm suggesting using the word community. It's in the woordenlijst from the Taalunie - http://woordenlijst.org/#/?q=community and is well-recognized by Dutch and Flemish people (91%,86% in 2013). So I think we're good there.